doc/api-hidden.rst

```diff
@@ -27,13 +27,66 @@
    Dataset.std
    Dataset.var
 
+   core.rolling.DatasetCoarsen
+
    core.groupby.DatasetGroupBy.assign
    core.groupby.DatasetGroupBy.assign_coords
    core.groupby.DatasetGroupBy.first
    core.groupby.DatasetGroupBy.last
    core.groupby.DatasetGroupBy.fillna
    core.groupby.DatasetGroupBy.quantile
    core.groupby.DatasetGroupBy.where
+   core.groupby.DatasetGroupBy.all
```

Comment:
I actually think these methods should not be hidden. They are visible if you navigates to `DataArrayGroupBy` and scroll down to find methods but that's totally not obvious.

Reply:
the other methods except from `map` and `reduce` are listed under `grouped operations`. I'd say increasing the visibility of these is out of scope and I'd like to leave that to a new PR / issue where we can have a focused discussion. The current changes just make sure a documentation page is created, so navigating to the `DataArrayGroupBy` page does not expose broken links.

Reply:
:+1: